### ISSUE TITLE
Unable to run with mina 1.0

### ISSUE THREAD

**Issue by pedroresende** [endoze/mina-multistage#21]
After installing mina-multistage with mina 1.0 when I run the mina command I get the following error

```
mina aborted!
NameError: undefined local variable or method `all_stages' for main:Object
/Library/Ruby/Gems/2.0.0/gems/mina-multistage-1.0.2/lib/mina/multistage.rb:36:in `<top (required)>'
/tmp/project/config/deploy.rb:5:in `<top (required)>'
/Library/Ruby/Gems/2.0.0/gems/mina-1.0.0/lib/mina/application.rb:16:in `run'
/Library/Ruby/Gems/2.0.0/gems/mina-1.0.0/bin/mina:4:in `<top (required)>'
/usr/local/bin/mina:23:in `load'
/usr/local/bin/mina:23:in `<main>'
(See full trace by running task with --trace)
```

**Comment by endoze** [comment]
I've just released version 1.0.3 that resolves this issue. Thanks for your contribution!

**Comment by pedroresende** [comment]
@endoze Thanks :+1:
